src/main/java/com/thealgorithms/datastructures/stacks/CalculateMaxOfMin.java

```diff
@@ -0,0 +1,43 @@
+/** Author : Siddhant Swarup Mallick
+ * Github : https://github.com/siddhant2002
+ */
+
+/** Program description - Given an integer array. The task is to find the maximum of the minimum of the array */
+package com.thealgorithms.datastructures.stacks;
+
+import java.util.*;
+
+public class CalculateMaxOfMin {
+    public static boolean calculateMaxOfMin(int[] a, int b[], int n) {
```

Comment:
The function signature doesn't correspond to its description. Please remove parameters b and n and return the max of min array instead of bool